### PR TITLE
Fix: Fix header on main chat room not aligned

### PR DESCRIPTION
**Pull request by Ginohmk**
- Fixed layout issue by aligning it alongside the message section layout, no need for it to be a position fixed, as people keep setting their desired top position value that breaks the code layout. @JulianaSau @ickynavigator @Maverickk6 

#### Before
Screenshot from 2022-11-27 12-09-35

#### After
image

##### Note:
- Next working: working on the comment layout that was scattered by a bad code merged into I and @Maverickk6  previous solution.

### CODE REVIEW

File: packages/root-config/src/assets/plugin.importmap.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
     "@zuri/zuri-plugin-tools": "https://externaltools.zuri.chat/zuri-zuri-plugin-tools.js",
     "@zuri/zuri-plugin-channels": "https://channels.zuri.chat/static/main/dist/zuri-zuri-plugin-channels.js",
     "@zuri/zuri-plugin-contributions": "https://ct.zuri.chat/js/app.js",
-    "@zuri/zuri-plugin-messaging": "https://chat.zuri.chat/zuri-messaging.js"
+    "@zuri/zuri-plugin-messaging": "http://localhost:5013/zuri-messaging.js"
```
Comment (billmal071) [inline packages/root-config/src/assets/plugin.importmap.json:17]: Change back to the live link.

File: packages/main/src/pages/protected/workspace/Workspace.style.js [hunk 2]
```diff
@@ -21,9 +21,9 @@ export const TopBarWrapperStyle = styled.div`
 `;
 
 export const SidebarWrapperStyle = styled.div`
-  flex-basis: 20%;
-  max-width: 300px;
-  min-width: 230px;
+  /* flex-basis: 20%; */
+  /* max-width: 300px;
+  min-width: 230px; */
```
Comment (billmal071) [inline packages/main/src/pages/protected/workspace/Workspace.style.js:26]: You can remove these.

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const Sidebar = props => {
 
   const sidebarRef = useRef(null);
   const [isResizing, setIsResizing] = useState(false);
-  const [sidebarWidth, setSidebarWidth] = useState(320);
+  const [sidebarWidth, setSidebarWidth] = useState(290);
 
   const startResizing = useCallback(() => {
```
Comment (billmal071) [inline packages/main/src/components/protected/sidebar/components/Sidebar.jsx:30]: Add the isRezing to the deps array.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by Ginohmk** [comment]
#### Before
Screenshot from 2022-11-27 13-33-08

##### After 
image

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx (hunk 3 above)
Comment (Ginohmk) [reply packages/main/src/components/protected/sidebar/components/Sidebar.jsx:30]: added @billmal071

File: packages/ui/src/comment-board/CommentBoard.styled.js [hunk 4]
```diff
@@ -4,28 +4,41 @@ const CommentBoardWrapper = styled.aside`
...
 const CommentBoardHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   background: hsla(160, 100%, 36%, 1);
   padding: 6px 16px;
   z-index: 99;
-  height: 40px;
+  height: 41px;
   /* margin-right: -16px; */
```
Comment (billmal071) [inline packages/ui/src/comment-board/CommentBoard.styled.js:29]: remove.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by billmal071** [comment]
Waiting for @Mavrik-jnr 's fixes

File: packages/root-config/src/assets/plugin.importmap.json (hunk 1 above)
Comment (billmal071) [reply packages/root-config/src/assets/plugin.importmap.json:17]: @Ginohmk

**Comment by Mavrik-jnr** [comment]
> Waiting for @Mavrik-jnr 's fixes

Due to my system currently down, we were on a videocall while he codes so we've refered to the changes you pointed earlier so he's currently making the pushes now please.

**Comment by Ginohmk** [comment]
- Final Push 

https://user-images.githubusercontent.com/58771507/204141024-94c7bf12-a6a5-4e95-978e-c17542261078.mp4

**Pull request merged**